assistant_dists/dream_emotion/docker-compose.override.yml

```diff
@@ -0,0 +1,478 @@
+services:
+  agent:
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.pipeline_config=assistant_dists/dream_emotion/pipeline_conf.json'
+    environment:
+      WAIT_HOSTS: "sentseg:8011,
+          dff-intent-responder-skill:8012, intent-catcher:8014, ner:8021,
+          factoid-qa:8071, kbqa:8072, entity-linking:8075, wiki-parser:8077, text-qa:8078,
+          combined-classification:8087, fact-retrieval:8100, entity-detection:8103,
+          sentence-ranker:8128, property-extraction:8136, prompt-selector:8135, openai-api-chatgpt:8145,
+          dff-dream-persona-chatgpt-prompted-skill:8137, dff-dream-faq-prompted-skill:8170,
+          openai-api-chatgpt-16k:8167, bot-emotion-classifier:8051, emotional-bot-response:8050,
+          emotion-ranking-based-response-selector:8048"
```

Comment:
i suggest to get rid of extra components in this distribution. For example, factoid compoentns, etc.
So, you me leave components from "in general" prompted distribution (aka https://github.com/deeppavlov/dream/tree/main/assistant_dists/dream_persona_openai_prompted):

sentseg:8011, combined-classification:8087, 
          sentence-ranker:8128,  prompt-selector:8135, openai-api-chatgpt:8145,
          dff-dream-persona-chatgpt-prompted-skill:8137, dff-dream-faq-prompted-skill:8170,
          openai-api-chatgpt-16k:8167, bot-emotion-classifier:8051, emotional-bot-response:8050,
          emotion-ranking-based-response-selector:8048

Reply:
done. Hope I removed unnecessary components from all config files...